Day6/github-api-project-ts/src/services/index.ts

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+import { IUser } from "../types";
+
+const endpoint = "https://api.github.com/users/";
+
+interface IUserDataResponse {
+  data: IUser;
+}
+
+const getUserData = (user: string): Promise<IUserDataResponse> =>
+  axios.get(`${endpoint}${user}`);
```

Comment:
try to make use of Promise.all or axios.all to make concurrent request for faster response.

Reply:
Done.